pkg/expression/util.go

```diff
@@ -1492,6 +1492,28 @@ func containMutableConst(ctx EvalContext, exprs []Expression) bool {
 	return false
 }
 
+// ReEvaluateParamMarkerConst updates all constants to their current ParamMarkers
+func ReEvaluateParamMarkerConst(ctx sessionctx.Context, exprs []Expression) error {
```

Comment:
Could we move this function into `plan_cache_utils.go` since it seems like only plan cache will use it?

Reply:
It seems like only the Plan Cache has the requirement to Re-Eval some expressions...